.appveyor/install.sh

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env bash
+
+export PATH="/mingw64/bin:${PATH}"
+export PKG_CONFIG_PATH="/mingw64/lib/pkgconfig:${PKG_CONFIG_PATH}"
+export GOROOT=/mingw64/lib/go
+export GOPATH=/go
+export CGO_ENABLED=1
+
+pacman --noconfirm -S \
+    mingw64/mingw-w64-x86_64-go \
+    mingw64/mingw-w64-x86_64-libusb
+
+go version
+go get github.com/google/gousb/...
+go test github.com/google/gousb/...
+go build -o gousb-win64.exe github.com/google/gousb
```

Comment:
the file in your repository does not have an "end of line" character at the end of the file. Github marks that in the "files changed" view, and my vim complains "[noeol]".

IntelliJ has an option to insert a linefeed at the end of the file, https://stackoverflow.com/questions/16761227/how-to-make-intellij-idea-insert-a-new-line-at-every-end-of-file. Presumably you have something like that in Gogland too?

Your Go files don't have this problem because gofmt adds an EOL char automatically. Only text/shell files are affected.